cmd/infracost/comment_azure_repos.go

```diff
@@ -132,8 +138,10 @@ func commentAzureReposCmd(ctx *config.RunContext) *cobra.Command {
 	cmd.Flags().String("azure-access-token", "", "Azure DevOps access token")
 	_ = cmd.MarkFlagRequired("azure-access-token")
 	cmd.Flags().StringArrayP("path", "p", []string{}, "Path to Infracost JSON files, glob patterns need quotes")
-	_ = cmd.MarkFlagRequired("path")
 	_ = cmd.MarkFlagFilename("path", "json")
+	cmd.Flags().String("comment-path", "", "Path to comment markdown file (experimental)")
```

Comment:
fwiw, this doesn't have to be markdown

Reply:
I agree, but I thought it's better to keep it consistent as markdown also supports HTML.